src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
```

Comment:
I would recommend to order import statements as [mentioned](https://mate-academy.github.io/style-guides/java/java.html#s3.3.3-import-ordering-and-spacing) in the guide.